pkgs/development/python-modules/mediapipe/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, ffmpeg
+, opencv4
+, numpy
+, cycler
+, fonttools
+, kiwisolver
+, matplotlib
+, pillow
+, protobuf
+, absl-py
+, pythonOlder
+, pythonAtLeast
+, python
+}:
+let
+  abi = "cp310";
+in
+buildPythonPackage rec{
+  pname = "mediapipe";
+  version = "0.10.5";
+  #Wheel has been compiled specifically for cython 3.10
+  disabled = pythonOlder "3.10" || pythonAtLeast "3.11" || python.implementation != "cpython";
+
+  src = fetchPypi {
+    inherit pname version format abi;
+    dist = abi;
+    python = abi;
+    platform = "manylinux_2_17_x86_64.manylinux2014_x86_64";
+    hash = "sha256-lBD8mgs00e3QC1/23fVP1JZIUn+n6G25+BdELEwIDV0=";
+  };
+
+  format = "wheel";
+  # otherwise opencv-contrib-python (aka opencv) will be missed
+  #yes, this does mean that every dependency requires the below line
+  pipInstallFlags = [ "--no-deps" ];
+  propagatedBuildInputs = [
+    ffmpeg
+    opencv4
+    numpy
+    cycler
+    fonttools
+    kiwisolver
+    matplotlib
+    pillow
+    protobuf
+    absl-py
+  ];
+
+  meta = with lib;{
```

Comment:
`sourceProvinance`, analogous to https://github.com/NixOS/nixpkgs/blob/e937f7ded66c4f2c3e97cbfdcd9dcc57cb648856/pkgs/development/python-modules/torch/bin.nix#L112

Also maybe consider renaming the attribute from `mediapipe` to `mediapipe-bin`, to leave room for the source build if anyone eventually decides to package it?